src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -443,7 +483,7 @@ public ItemViewModel(LayoutPreferencesManager folderSettingsViewModel)
 			operationEvent = new AsyncManualResetEvent();
 			gitChangedEvent = new AsyncManualResetEvent();
 			enumFolderSemaphore = new SemaphoreSlim(1, 1);
-			loadPropertiesSemaphore = new SemaphoreSlim(100);
+			getFileOrFolderSemaphore = new SemaphoreSlim(50);
```

Comment:
The app sometimes crashed in my environment even at 100, so I reduced it.

Reply:
Seems good to me